go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/Azure/go-autorest/autorest v0.9.0
 	github.com/Azure/go-autorest/autorest/adal v0.5.0
 	github.com/Azure/go-autorest/autorest/to v0.2.0
-	github.com/GoogleCloudPlatform/k8s-cloud-provider v0.0.0-20190822182118-27a4ced34534
+	github.com/GoogleCloudPlatform/k8s-cloud-provider v0.0.0-20200203062548-7bdc6cbb875a
```

Comment:
this is still at a random sha... is this supposed to be v1.12.0?

Reply:
right `hack/pin-dependency.sh github.com/GoogleCloudPlatform/k8s-cloud-provider 1.12.0` still uses the SHA. so i added a comment to remind us.

Reply:
is there a reason upstream didn't use a semver tag? v1.12.0 would have been better